pkg/trace/api/otlp.go

```diff
@@ -325,6 +330,9 @@ func (o *OTLPReceiver) createChunks(tracesByID map[uint64]pb.Trace, prioritiesBy
 			Tags:  map[string]string{"_dd.otlp_sr": rate},
 			Spans: spans,
 		}
+		if tid, ok := tidByID[k]; ok {
+			traceutil.SetMeta(spans[0], "_dd.p.tid", tid)
```

Comment:
We already have `otel.trace_id` on [each and every span](https://github.com/DataDog/datadog-agent/blob/055d626c2415fa37ffb9968b6208eef52a3a8338/pkg/trace/api/otlp.go#L451). Why do we need to duplicate it in another tag (despite the fact that we cut off the second half)? Can't the backend just pick it up? This seems like unnecessary complexity...